pip_chill/pip_chill.py

```diff
@@ -16,8 +16,8 @@ def chill(show_all=False):
     # Gather all packages that are requirements and will be auto-installed.
     distributions = {}
     dependencies = {}
-
-    for distribution in pip.get_installed_distributions():
+    
+    for distribution in pkg_resources.working_set:
```

Comment:
Replacing `pip.get_installed_distributions` with `pkg_resources.working_set` is a better fix than importing `pip._internal.utils.misc.get_installed_distributions` as it's *internal* as the name say.

However this means this PR should also add `setuptools` to the `requirements` list in `setup.py`

`pip` should be there as well right now but isn't, even though it's actually used in the code, not just in tests.

If `pip` was there, this PR should replace `pip` with `setuptools`.

Reply:
Another approach is to use `pip-shims` or `pip-api`